rllib/algorithms/sac/sac_rl_module.py

```diff
@@ -54,6 +56,11 @@ class SACRLModule(RLModule, InferenceOnlyAPI, TargetNetworkAPI):
 
     @override(RLModule)
     def setup(self):
+        # TODO (sven): Temporary fix (until we have figured out the final config
+        #  architecture for default models). If self.model_config is a dict (which
+        #  it should always be, make sure to merge it with the default config).
+        self.model_config = dataclasses.asdict(DefaultModelConfig()) | self.model_config
```

Comment:
I guess we need to add this also to the `BCRLModule` then.

Reply:
Also: We might need to add this to all the examples that are not derived from the algorithm modules.

Reply:
Great catch! Done ...

Reply:
For the non-default models, this is fine, b/c those are 100% user-driven and thus don't use the catalogs. The user decides, which keys are accessed in the `model_config` and only those need to be provided (or even have default values inside the custom RLModule code).

This is really only a problem for RLlib's default models.